packages/headless/src/features/search/search-request.ts

```diff
@@ -70,10 +70,12 @@ export const buildSearchRequest = async (
         currentFacets: automaticFacets,
       },
     }),
-    ...(state.generatedAnswer?.responseFormat && {
+    ...(state.generatedAnswer && {
       pipelineRuleParameters: {
         mlGenerativeQuestionAnswering: {
           responseFormat: state.generatedAnswer.responseFormat,
+          citationsFieldToInclude:
```

Comment:
`field` is singular here?

Reply:
Yeah that's how they named the parameter.